modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,31 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+You can create a secret manually by running the following command:
+[source,terminal,subs="+quotes"]
+----
+oc -n openshift-logging create secret generic "logging-loki-s3"
+--from-literal=bucketnames="<s3_bucket_name>"
+--from-literal=region="<bucket_region>"
```

Comment:
Just to keep things uniform here too, I think the secret name should be kept the same. 

```
oc -n openshift-logging create secret generic "logging-loki-aws"
--from-literal=bucketnames="<s3_bucket_name>"
--from-literal=region="<bucket_region>"
```